net-p2p/cardano-node/Makefile

```diff
@@ -48,7 +48,7 @@ BINARY_ALIAS=		git=true
 USERS=		cardano
 GROUPS=		cardano
 
-USE_CABAL=	Diff-0.4.1 \
+USE_CABAL=      Diff-0.4.1 \
```

Comment:
Is this a space?

Reply:
Of course it is... how did it get there...

Reply:
fixed